components/ui/spinner.tsx

```
@@ -0,0 +1,68 @@
+        {Array.from({ length: 8 }).map((_, i) => (
+          <span
+            key={i}
+            className="absolute top-0 left-1/2 w-[12.5%] h-full animate-spinner-leaf-fade"
+            style={{
+              transform: `rotate(${i * 45}deg)`,
+              animationDelay: `${-(7 - i) * 100}ms`,
+            }}
+          >
+            <span
+              className={cn("block w-full h-[30%] rounded-full", bgColorClass)}
+            ></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)